Doc/library/json.rst

```diff
@@ -738,6 +738,12 @@ Command line options
 
    .. versionadded:: 3.8
 
+.. cmdoption:: --indent, --tab, --no-indent, --compact
+
+   Mutually exclusive options for whitespace control
+
+   .. versionadded:: 3.9
+
```

Comment:
@wimglenn does this documentation look okay? I combined multiple arguments into a single line (not sure if that violates convention or is okay)